apps/dashboard/app/javascript/packs/support_ticket.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
+
+jQuery(function (){
+    const EMAIL_REGEX = /^([\w-\.\+]+@([\w-]+\.)+[\w-]{2,4})?$/;
```

Comment:
OK - I found that html5 validations do already respond to localizations. Here's french (note the hove-rover text box too).

![image](https://user-images.githubusercontent.com/4874123/191077911-f3867709-a2c3-40d4-a6b5-b20171525d88.png)

I know we can't do _everything_ with html5 validation - but I'd like to get most simple things like email just for simplicity's sake.

Reply:
OK - Will move as much front end validation as possible to HTML5